client/components/tooltip/tooltip-base.tsx

```diff
@@ -112,28 +137,40 @@ const TooltipPortal = memo( ( { children } ) => {
 	// on component unmount, clear any reference to the created node
 	useEffect( () => {
 		return () => {
-			rootElement.removeChild( node.current );
-			node.current = null;
+			if ( node.current ) {
+				rootElement.removeChild( node.current );
+				node.current = null;
+			}
```

Comment:
Moved this within an if statement to confirm `node.current` exists before removing from the DOM.